src/main/java/fr/socrates/domain/candidate/CandidateServiceImpl.java

```diff
@@ -17,10 +17,21 @@ public CandidateServiceImpl(CandidateRepository candidateRepository) {
     }
 
     @Override
-    public boolean add(Candidate candidate) {
+    public boolean add(Candidate candidate) throws CandidatePersisteDataException, CandidateExistingException {
+        checkThatCandidateDoesntExist(candidate);
+        checkSuccessfulCandidateSave(candidate);
+        return true;
+    }
+
+    private void checkSuccessfulCandidateSave(Candidate candidate) throws CandidatePersisteDataException {
+        final boolean successfulSave = candidateRepository.save(candidate);
+        if (!successfulSave)
```

Comment:
Can you add braces even for one line if statement?
We should discuss that kind of code rules